tapback/backend.c

```diff
@@ -742,7 +742,7 @@ tapback_backend_probe_device(backend_t *backend,
      * We don't set a XenStore watch on these paths in order to limit the
      * number of watches for performance reasons.
      */
-    if (!remove && comp) {
+    if (!remove && comp && device) {
```

Comment:
I do not see a path how `&& device` would be false with `!remove` being checked before.

Reply:
tapback_backend_find_device is actually a macro. This code path is triggered during iso hotswap when the device is NULL for a short amount of time.

Reply:
I got it now, all good, thanks.